docs/tutorials/mocks/mock-dependencies.md

```diff
@@ -328,12 +328,12 @@ that the bug in `CreateAccount` is now triggered and the assertion fails!
 Awesome! Using `Task.Run` in the mock methods introduces concurrency in the test, which allows the
 two `CreateAccount` methods to execute asynchronously and race with each other. This is similar to
 how invoking the production implementation of `IDbCollection` (i.e. the actual backend NoSQL
-database) typically happens asynchronously, so network calls do not block the rest of the system.
+database) typically happens asynchronously.
 
 Can you make the above mock a little more generally applicable, so you don't have to write custom
 mocks for each test case? What if you model it in a way that more closely simulates the behavior of
-the actual `IDbCollection`? Let's write such a mock that we can use in concurrency unit tests for
-the `AccountManager`.
+the actual `IDbCollection`? You could a mock that you can use in all your concurrency unit tests for
```

Comment:
There is a typo here, should be `You could write a mock`` (write is missing).

Reply:
@lovettchris, just fixed it myself to lower your overhead!

Reply:
thanks.